llvm/tools/llvm-profgen/ProfileGenerator.cpp

```diff
@@ -83,6 +83,10 @@ static cl::opt<double> HotFunctionDensityThreshold(
 static cl::opt<bool> ShowDensity("show-density", llvm::cl::init(false),
                                  llvm::cl::desc("show profile density details"),
                                  llvm::cl::Optional);
+static cl::opt<int> ProfileDensityHotFuncCutOff(
+    "profile-density-hot-func-cutoff", llvm::cl::init(990000),
```

Comment:
Do we need a new switch? Should we use `ProfileSummaryCutoffHot` instead given they are the same value?

Reply:
This is just to in case people only want to tune the the profile density threshold(like to reduce the noise) but not the `ProfileSummaryCutoffHot` which will affect the profile generation(pre-inliner).